### PR TITLE
hw6.2

### PR DESCRIPTION
**Pull request by Mykhaylo12**

### CODE REVIEW

File: src/main/java/core/basesyntax/impl/StorageImpl.java [hunk 1]
```diff
@@ -1,15 +1,18 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.HashMap;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    HashMap<K, V> storage = new HashMap<>();
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/impl/StorageImpl.java]: we shouldn't use hashmap for this task

File: src/main/java/core/basesyntax/impl/StorageImpl.java [hunk 2]
```diff
@@ -3,13 +3,28 @@
...
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_SIZE = 10;
+    private Object[] keyStorage = new Object[ARRAY_SIZE];
+    private Object[] valueStorage = new Object[ARRAY_SIZE];
 
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < valueStorage.length; i++) {
+            if (valueStorage[i] == null & keyStorage[i] == null | keyStorage[i] == key) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/impl/StorageImpl.java]: values == null are allowed and keys should be compared by equals so this if should be fixed :)

File: src/main/java/core/basesyntax/impl/StorageImpl.java (hunk 1 above)
Comment (Mykhaylo12) [reply src/main/java/core/basesyntax/impl/StorageImpl.java]: ok

File: src/main/java/core/basesyntax/impl/StorageImpl.java (hunk 2 above)
Comment (Mykhaylo12) [reply src/main/java/core/basesyntax/impl/StorageImpl.java]: ok

**Pull request closed without merging**